libexec/bats

```diff
@@ -20,6 +20,8 @@ help() {
   echo "  -h, --help     Display this help message"
   echo "  -p, --pretty   Show results in pretty format (default for terminals)"
   echo "  -t, --tap      Show results in TAP format"
+  echo "  -x, --extended Show results in extended TAP format"
+  echo "  -u, --junit    Show results in JUnit format, hudson compatible"
```

Comment:
Better to say Jenkins or Jenkins/Hudson here as most people use Jenkins. Additionally it's a personal name so caps are grammatically correct.